src/routes/_surfaces/workspace/explore/leaderboards/DimensionDisplay.svelte

```diff
@@ -61,15 +73,20 @@
     !$metaQuery.isRefetching
   ) {
     topListQuery = useTopListQuery(metricsDefId, dimensionId, {
-      measures: metricsExplorer?.selectedMeasureIds,
+      measures: $selectedMeasureNames.data,
```

Comment:
Here, "selected" measures are really "all" measures, right? When I read "selected", I think of client-side state that would be coming from the Svelte Store. But here it looks like we're getting *all* (valid?) measures from server-side. Is that right?